<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwndCtrlPannel</class>
 <widget class="QMainWindow" name="mainwndCtrlPannel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mainwndCtrlPannel</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget_main">
      <property name="tabPosition">
       <enum>QTabWidget::South</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <widget class="QWidget" name="tab_summary">
       <attribute name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/pkCtrlPannel/resources/services-2.png</normaloff>:/pkCtrlPannel/resources/services-2.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Summary</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Parkinglots</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableView" name="listView_sum_pklts">
              <property name="statusTip">
               <string>Double click item to see details.</string>
              </property>
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
              <property name="sortingEnabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>MACs</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableView" name="listView_sum_macs">
              <property name="statusTip">
               <string>Double click item to see details.</string>
              </property>
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
              <property name="sortingEnabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Devices</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableView" name="listView_sum_devices">
            <property name="statusTip">
             <string>Double click item to see details.</string>
            </property>
            <property name="alternatingRowColors">
             <bool>false</bool>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_detail">
       <attribute name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/pkCtrlPannel/resources/Documents-12.png</normaloff>:/pkCtrlPannel/resources/Documents-12.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Detail</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Detailed View</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QTableView" name="tableView_detailed">
             <property name="statusTip">
              <string>Double click item to see details.</string>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Events History</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QTableView" name="tableView_events">
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_macevt">
       <attribute name="title">
        <string>MAC Events</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QTableView" name="tableView_macevt">
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_ctrl">
       <attribute name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/pkCtrlPannel/resources/AquaBubble 037.png</normaloff>:/pkCtrlPannel/resources/AquaBubble 037.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Control</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QTabWidget" name="tabWidget_ctrl">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_target">
           <attribute name="title">
            <string>Target</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Server IP</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_SvrIP"/>
              </item>
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Server Port</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_Svr_Port"/>
              </item>
              <item>
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>Tmp ID:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_tmpid">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Target Mac ID</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_tarID_Mac"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>Target Dev ID</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_tarID_Dev"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QPushButton" name="pushButton_getMacInfo">
                <property name="text">
                 <string>Get Mac Info</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_getDevList">
                <property name="text">
                 <string>Get Dev List</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_getDevPara">
                <property name="text">
                 <string>Get Dev Para</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButton_Mac">
                <property name="text">
                 <string>Mac</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButton_Dev">
                <property name="text">
                 <string>Device</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>Get Info</string>
           </attribute>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Return message</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit_result"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_express">
       <attribute name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/pkCtrlPannel/resources/havecar.png</normaloff>:/pkCtrlPannel/resources/havecar.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Express</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QListView" name="listView_express">
          <property name="iconSize">
           <size>
            <width>128</width>
            <height>128</height>
           </size>
          </property>
          <property name="movement">
           <enum>QListView::Snap</enum>
          </property>
          <property name="spacing">
           <number>16</number>
          </property>
          <property name="viewMode">
           <enum>QListView::IconMode</enum>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>539</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Service">
    <property name="title">
     <string>&amp;Service</string>
    </property>
    <addaction name="actionConfig"/>
    <addaction name="action_Connect"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <widget class="QMenu" name="menu_View_2">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="action_Refresh"/>
   </widget>
   <addaction name="menu_Service"/>
   <addaction name="menu_View_2"/>
   <addaction name="menu_View"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Connect"/>
   <addaction name="action_Refresh"/>
   <addaction name="separator"/>
   <addaction name="actionConfig"/>
   <addaction name="action_About"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Connect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/pkCtrlPannel/resources/0089.png</normaloff>:/pkCtrlPannel/resources/0089.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Connect</string>
   </property>
  </action>
  <action name="actionConfig">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/pkCtrlPannel/resources/111.png</normaloff>:/pkCtrlPannel/resources/111.png</iconset>
   </property>
   <property name="text">
    <string>Con&amp;fig</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="action_About">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/pkCtrlPannel/resources/chrysler_300m.png</normaloff>:/pkCtrlPannel/resources/chrysler_300m.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_Refresh">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/pkCtrlPannel/resources/services-2.png</normaloff>:/pkCtrlPannel/resources/services-2.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Refresh</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
